pkg/types/status.go

```diff
@@ -20,18 +23,21 @@ var (
 		"will_not_fix",
 		"fix_deferred",
 		"end_of_life",
+		"needed",
+		"pending",
 	}
 )
 
 const (
 	StatusUnknown Status = iota
 	StatusNotAffected
 	StatusAffected
-	StatusFixed
+	StatusFixed // Ubuntu "Released" Mapped to Fixed
 	StatusUnderInvestigation
-	StatusWillNotFix // Red Hat specific
+	StatusWillNotFix // Red Hat specific // Ubuntu Specific "ignored" Superseded || End of life || End of Kernal support
 	StatusFixDeferred
 	StatusEndOfLife
+	StatusPending //Ubuntu Specific "pending","needed"
```

Comment:
```suggestion
	StatusPending
```
I've suggested above this state can just be used as `StatusFixDeferred`, but if there is a reason we need extra states here, as this is also used by Red Hat IIRC, it makes sense to just remove the comment, as this state can be used by multiple providers.